lib/fetchers/mdl_api_fetcher.py

```diff
@@ -0,0 +1,191 @@
+import json
+from dplaingestion.fetchers.fetcher import *
+
+class MDLAPIFetcher(Fetcher):
+    def __init__(self, profile, uri_base, config_file):
+        super(MDLAPIFetcher, self).__init__(profile, uri_base, config_file)
+        self.total_records = None
+        self.endpoint_url_params = profile.get("endpoint_url_params")
+
+    def extract_content(self, content, url):
+        error = None
+        try:
+            parsed_content = json.loads(content)
+        except Exception, e:
+            error = "Error parsing content from URL %s: %s" % (url, e)
+            return error, content
+
+        content = parsed_content.get("response")
+
+        if not self.total_records:
+            total_records_prop = "numFound"
+            self.total_records = getprop(content, total_records_prop)
+
+        if content is None:
+            error = "Error, there is no \"response\" field in content from " \
+                    "URL %s" % url
+        # elif exists(content, "response/headers/code"):
```

Comment:
Should this be commented out?

If this action should not be taken, would it be more clear with a comment above saying what's not going to be done?